test/models/vote_test.rb

```diff
@@ -0,0 +1,9 @@
+require "test_helper"
+
+describe Vote do
+  let(:vote) { Vote.new }
+  let(:user) { User.new(username: 'Amanda')}
+  it "must be valid" do
+    value(vote).must_be :valid?
```

Comment:
This test is failing, because you didn't use a fixture, you just did `Vote.new`, but this looks like the Rails generated test.